job_export.py

```diff
@@ -18,6 +18,8 @@
 from pelican.s3 import s3upload_file
 from pelican.indexd import indexd_submit
 from pelican.mds import metadata_submit_expiration
+from pfb_to_zip.pfb_to_zip import PFBExporter
+import pfb_to_zip
```

Comment:
do these 2 need to be a different import? 

Since you are already doing `import pfb_to_zip` could you just reference the same one as pfb_to_zip.pfb_to_zip. PFBExporter() when you use it?

Reply:
since the package name is "pfb_to_zip" and the file name inside the package is "pfb_to_zip" we cant implicitly import PFBExporter via:
"import pfb_to_zip"
" var = pfb_to_zip.pfb_to_zip.PFBExporter()" since pfb_to_zip.pfb_to_zip is pointing to the init file as pfb_to_zip

so we either have to leave it or rename the file pfb_to_zip.py to something else

Reply:
@grugna wait let me rephrase this that was no correct. but lines have to be there since we need to import the package "import pfb_to_zip" in order to gather the path variable to the config files since the configs file are being opened not imported. but we can not from there run pfb_to_zip.pfb_to_zip.PFB_exporter because this part "pfb_to_zip.PFB_exporter" is not being import in the __init__ file. The portion about the file name having to change is if you want to import PFB_exporter into the __init__ file then you can just add a "from .pfb_to_zip" to specify the file not the package 
![Screenshot 2025-05-05 at 10 25 27 AM](https://github.com/user-attachments/assets/4c04b8bd-883b-4f31-99fa-c87af5be8887)